docs/vscode.md

```diff
@@ -0,0 +1,105 @@
+# Visual Studio Code Installation
+
+Install nodeJs, yarn and java according to your system:
+- Windows: [./windows-setup.md](./windows-setup.md)
+- Linux / Windows Subsystem for Linux: [./wsl-setup.md](./wsl-setup.md)
+
+Download and Install Visual Studio Code or use chocolatey:
+
+https://code.visualstudio.com/Download  
+https://code.visualstudio.com/docs/setup/linux  
+or
+```
+choco install vscode
+```
+
+## Settings
+
+Adjust the following settings corresponding to your preferences: 
+```
+File > Preferences > Keyboard Shortcuts
+# or install a known ones (e.g. IntelliJ)
+File > Preferences > Keymaps
+```
+
+### Optional settings
+```
+File > Preferences > Settings (Ctrl+,)
+```
+```
+// Do not close file when opening another one
+"workbench.editor.enablePreviewFromQuickOpen": false,    
+"workbench.editor.enablePreview": false,
+
+// Do not show mini code preview on the right side
+"editor.minimap.enabled": false,
+
+// Do not automatically commmit unstaged files
+"git.enableSmartCommit": false,
+
+// Privacy
+"telemetry.enableCrashReporter": false,
+"telemetry.enableTelemetry": false,
+"update.enableWindowsBackgroundUpdates": false
+```
+
+## Required Extensions
+
+The following extensions (or any alternatives) should be installed.
+
+### Flow Type Checker 
+(Alternative: Flow Language Support)
+
+Hint: Use "yarn flow check" instead of "yarn flow" in the console, currently there is a [Bug](https://github.com/facebook/flow/issues/6592) in Windows
```

Comment:
```suggestion
Hint: Use "yarn flow check" instead of "yarn flow" in the console, currently there is a [Bug](https://github.com/facebook/flow/issues/6592) on Windows
```